docs/approvals/index.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Approvals
+description: Approvals, approvals, approvals
+position: 800
```

Comment:
After the `position` element, I'd recommend putting:

```yml
hideInThisSection: true
```
That would prevent another link to Service Now from being rendered
![image](https://user-images.githubusercontent.com/1418993/169818067-4bbba628-81ba-4877-bfb7-e1c3183d2cea.png)